### ISSUE TITLE
How to train the recowith the pretrained model

### ISSUE THREAD

**Issue by FengMu1995** [deepinsight/insightface#1998]
I would like to finetune the arcface model with , but the softmax weight

**Comment by dhagarwa** [comment]
+1

**Comment by BigJoon** [comment]
Any progress?

**Comment by skybocai** [comment]
Any progress?
